app/controllers/api/v1/passwords_controller.rb

```
@@ -0,0 +1,20 @@
+  before_action :set_user
+
+  def update
+    if @user.update(user_params)
+      render json: @user
+    else
+      render json: @user.errors, status: :unprocessable_entity
+    end
+  end
+
+  private
+    def set_user
```

RuboCop (1.76.1)

[convention] Use 2 (not 4) spaces for indentation.

(Layout/IndentationWidth)